src/leader_scheduler.rs

```diff
@@ -18,9 +18,10 @@ use solana_sdk::vote_transaction::VoteTransaction;
 use std::io::Cursor;
 use std::sync::Arc;
 
-pub const TICKS_PER_BLOCK: u64 = 4;
+pub const TICKS_PER_BLOCK: u64 = 128;
```

Comment:
why is this so large all of a sudden?  this will change confirmation time to ~2min?

Reply:
Leader rotation used to happen every TICKS_PER_BLOCK * DEFAULT_BLOCKS_PER_SLOT ticks, we changed DEFAULT_BLOCKS_PER_SLOT in this PR from 32 to 1, so TICKS_PER_BLOCK has to go up by a factor of 32 to compensate.

Reply:
Per offsite discussion, TICKS_PER_BLOCK and BLOCKS_PER_SLOT should no longer exist.  It should be TICKS_PER_SLOT only.  Blocks per slot will be 0 or 1 and should be generally useless for leader rotation.

Reply:
Votes in the replay_stage are currently generated every TICKS_PER_BLOCK, should they now be generated every TICKS_PER_SLOT? Are we only voting at the end of slots now?

Reply:
there is exactly one block per slot

Reply:
Ah ok yeah that agrees with what I understood from the offsite discussion, I missed the part where we vote on every slot. Thanks!

Reply:
More precisely, there is at most one block per slot. Validators vote on the last tick entry per slot. That last tick entry's id of a slot is also called a block id.